actors/builtin/miner/miner_test.go

```diff
@@ -3131,6 +3131,43 @@ func TestReportConsensusFault(t *testing.T) {
 		assert.Equal(t, reportEpoch+miner.ConsensusFaultIneligibilityDuration, endInfo.ConsensusFaultElapsed)
 	})
 
+	t.Run("Double report of consensus fault fails", func(t *testing.T) {
+		rt := builder.Build(t)
+		actor.constructAndVerify(rt)
+		precommitEpoch := abi.ChainEpoch(1)
+		rt.SetEpoch(precommitEpoch)
+		dealIDs := [][]abi.DealID{{1, 2}, {3, 4}}
+
+		startInfo := actor.getInfo(rt)
+		assert.Equal(t, abi.ChainEpoch(-1), startInfo.ConsensusFaultElapsed)
+		sectorInfo := actor.commitAndProveSectors(rt, 2, defaultSectorExpiration, dealIDs)
+		_ = sectorInfo
+
+		reportEpoch := abi.ChainEpoch(333)
+		rt.SetEpoch(reportEpoch)
+
+		actor.reportConsensusFault(rt, addr.TestAddress)
+		endInfo := actor.getInfo(rt)
+		assert.Equal(t, reportEpoch+miner.ConsensusFaultIneligibilityDuration, endInfo.ConsensusFaultElapsed)
+
+		rt.ExpectAbortContainsMessage(exitcode.ErrForbidden, "miner has existing fault that has not expired", func() {
+			actor.reportConsensusFault(rt, addr.TestAddress)
+		})
+		rt.Reset()
+
+		// new consensus faults are forbidden until original has elapsed
+		rt.SetEpoch(endInfo.ConsensusFaultElapsed) // actor.reportConsensusFault reports a fault for previous epoch
+		rt.ExpectAbortContainsMessage(exitcode.ErrForbidden, "miner has existing fault that has not expired", func() {
+			actor.reportConsensusFault(rt, addr.TestAddress)
+		})
+		rt.Reset()
+
+		// a new consensus fault can be reported once fault has elapsed
```

Comment:
Test that an old consensus fault cannot be reported (you'll need to mock the returned `Fault.Epoch`).